src/components/Layout/SearchList.css

```diff
@@ -0,0 +1,6 @@
+.search-results-header {
```

Comment:
can you use [BEM](https://getbem.com/naming/) format: like `search-results__header`